README.md

```diff
@@ -200,6 +200,7 @@ APIs
 - [Cambridge Dictionaries Online](http://dictionary.cambridge.org/license.html) - Access to Cambridge’s custom-developed API. 💸
 - [DictService](http://services.aonaware.com/DictService/DictService.asmx) - This Web Service allows you to access dictionary definitions from the dict.org service.
 - [Glosbe API](https://glosbe.com/a-api) - Dictionary API and Translation Memory API description.
+- [Lingua Robot API](https://www.linguarobot.io/) - Definition of words, pronunciations, synonyms and antonyms, lemmatization, morpheme segmentation. 💸
```

Comment:
`Definition of words, pronunciations, synonyms and antonyms, lemmatization, and morpheme segmentation.`

Reply:
Thanks, I've updated the description